src/app/modules/color-scheme/color-picker-launcher/color-picker-launcher.component.html

```diff
@@ -1,9 +1,9 @@
 <div>
-    <div class="launch-button" [class.circle]="layer.colorScheme.type == 'discrete'" [style.background]="layer.background" (click)="schemepopup.open()"></div>
+  <div [ngClass]="className" [style.background]="layer.background" (click)="schemepopup.open()"></div>
```

Comment:
No ngClass! There are 3 versions of the [class]:
```
1. [class.name]="true|false"
2. [class]="classes" where classes=[cls1, cls2, ...]
3. [class]="classes" where classes={cls1: true|false, cls2: true|false, ...}
```
 Case 2 is probably what you want here.